modules/oadp-installing-dpa.adoc

```diff
@@ -114,6 +128,13 @@ spec:
         name: default
         provider: {provider}
 ----
+<.> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that an environment with many nodes will have a lot of `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
```

Comment:
`Restic deploys a daemon set`  --> `Restic deploys a DaemonSet`

Reply:
`which means that an environment with many nodes will have a lot of `Restic` pods running.`
-->
`which means that an environment with each worker nodes will have a `Restic` pods running.`

Reply:
> `Restic deploys a daemon set` --> `Restic deploys a DaemonSet`

No, this is not [OpenShift style](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references).